card.status.req.notecard.api.json

```diff
@@ -4,6 +4,8 @@
     "title": "card.status Request Application Programming Interface (API) Schema",
     "description": "Returns general information about the Notecard's operating status.",
     "type": "object",
+    "version": "0.1.1",
```

Comment:
Roll this version, so we know we've put hands on it.

Then when we think we are actually finished, we can move it to `1.1.1`.